build/ci-build.yml

```diff
@@ -25,7 +25,7 @@ pool:
   vmImage: 'ubuntu-latest'
 
 steps:
-- template: 'pypi/determine-pr-version.yml@templates'
+- template: 'determine-build-version.yml'
```

Comment:
Is this to mitigate https://github.com/arcus-azure/azure-devops-templates/pull/40?

Reply:
yes, indeed - and will now adapt :)